lib/sync/db-templates/materializedViews.template.sql

```diff
@@ -274,3 +274,27 @@ AS (
             AND most_lenient_override.assignment_id = <%= externalSchema %>.submission_fact.assignment_id
     WHERE <%= externalSchema %>.assignment_dim.workflow_state = 'published'
 );
+
+DROP TABLE IF EXISTS <%= boacAnalyticsSchema %>.user_course_scores;
+CREATE TABLE <%= boacAnalyticsSchema %>.user_course_scores
+SORTKEY (course_id)
+AS (
+    SELECT
+        <%= externalSchema %>.user_dim.canvas_id AS user_id,
+        <%= externalSchema %>.course_dim.canvas_id AS course_id,
+        <%= externalSchema %>.course_score_fact.current_score AS current_score,
+        <%= externalSchema %>.course_score_fact.final_score AS final_score
+    FROM
+        <%= externalSchema %>.enrollment_fact
+        JOIN <%= externalSchema %>.enrollment_dim
+            ON <%= externalSchema %>.enrollment_fact.enrollment_id = <%= externalSchema %>.enrollment_fact.enrollment_id
```

Comment:
@paulkerschen `enrollment_dim. enrollment_id` needed in line above?

Reply:
Ack! Thank you, @johncrossman, amended.